knexfile.js

```diff
@@ -32,8 +32,8 @@ module.exports = {
     client: 'pg',
     connection: {
       host: 'localhost',
-      database: 'recursion',
-      user:     'recursion'
+      database: 'postgres',
```

Comment:
Basically these are credentials that the db recreator script needs to create and drop the test db.
On your machine it would be your username and no password etc...

So it was set to recursion for my local machine.
Now its set to postgres because thats what travis uses.
After this merge we will .gitignore the .knexfile and keep local copies and let travis/deploy use the main file.